expression/builtin_other_test.go

```diff
@@ -57,24 +60,26 @@ func (s *testEvaluatorSuite) TestBitCount(c *C) {
 	}
 	for _, test := range bitCountCases {
 		in := types.NewDatum(test.origin)
-		f, err := fc.getFunction(s.ctx, s.datumsToConstants([]types.Datum{in}))
-		c.Assert(err, IsNil)
-		c.Assert(f, NotNil)
+		f, err := fc.getFunction(ctx, datumsToConstants([]types.Datum{in}))
+		require.NoError(t, err)
+		require.NotNil(t, f)
 		count, err := evalBuiltinFunc(f, chunk.Row{})
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 		if count.IsNull() {
-			c.Assert(test.count, IsNil)
+			require.Nil(t, test.count)
 			continue
 		}
 		sc := new(stmtctx.StatementContext)
 		sc.IgnoreTruncate = true
 		res, err := count.ToInt64(sc)
-		c.Assert(err, IsNil)
-		c.Assert(res, Equals, test.count)
+		require.NoError(t, err)
+		require.Equal(t, test.count, res)
 	}
 }
 
-func (s *testEvaluatorSuite) TestInFunc(c *C) {
+func TestInFunc(t *testing.T) {
+	t.Parallel()
```

Comment:
we cannot make it parallel because of there is `collate.SetNewCollationEnabledForTest(true)` at L134

Reply:
i created a issue https://github.com/pingcap/tidb/issues/29319 to track this and hope correct all of this kind issue in one pr. could you please comment there to provide more detail about why call that functin the tests should run in serial?